qemu/tests/balloon_check.py

```diff
@@ -23,21 +23,23 @@ def __init__(self, test, params, env):
         super(BallooningTest, self).__init__(test, params, env)
 
         self.vm = env.get_vm(params["main_vm"])
-        self.session = self.get_session(self.vm)
-        self.params["balloon_test_setup_ready"] = False
-        if self.params.get('os_type') == 'windows':
-            sleep_time = 180
-        else:
-            sleep_time = 90
-        logging.info("Waiting %d seconds for guest's applications up" % sleep_time)
-        time.sleep(sleep_time)
-        self.params["balloon_test_setup_ready"] = True
+        if params.get("paused_after_start_vm") != "yes":
+            self.session = self.get_session(self.vm)
```

Comment:
I don't find any usage of this session, so might be dropped or at least close it some where.